spec/views/users/index.html.erb_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.describe "users/index", type: :view do
+  before(:each) do
+    assign(:users, [
+      User.create!(
+        username: "Username",
+        password: "Password",
+        aws_key: "Aws Key",
+        aws_secret: "Aws Secret"
+      ),
+      User.create!(
+        username: "Username",
+        password: "Password",
+        aws_key: "Aws Key",
+        aws_secret: "Aws Secret"
+      )
+    ])
+  end
+
+  it "renders a list of users" do
+    render
+    cell_selector = Rails::VERSION::STRING >= '7' ? 'div>p' : 'tr>td'
```

Comment:
Rails version here is 7, but you have changed to 6 in the gems.
Altough I suggested to maintain Rails as 7 and upgrade Mongoid to 8

Reply:
Mongoid version is already 8, as I already commented out in https://github.com/Fermented-Software/Kimchi/pull/27#discussion_r1002811001, I had some issues with the Gemfile.lock file but now i'm upgrading rails to version 7 in our app.